test/EFTools/UnitTests/EntityDesignerVersioningFacade/ReverseEngineerDb/SchemaDiscovery/EntityStoreSchemaQueryGeneratorTests.cs

```diff
@@ -283,7 +327,36 @@ public void Where_clause_created_for_single_Exclude_filter()
         [Fact]
         public void Where_clause_uses_AND_to_connect_multiple_aliases_and_Exclude_filter()
         {
-            var parameters = new EntityCommand().Parameters;
+            var parameters = CreateParameters();
+
+            Assert.Equal(
+                "NOT ((alias1.Name LIKE @p0))\r\nAND\r\nNOT ((alias2.Name LIKE @p1))",
+                new EntityStoreSchemaQueryGenerator(
+                    string.Empty,
+                    string.Empty,
+                    EntityStoreSchemaFilterObjectTypes.Table,
+                    new[]
+                        {
+                            new EntityStoreSchemaFilterEntry(
+                                null,
+                                null,
+                                "name",
+                                EntityStoreSchemaFilterObjectTypes.Table,
+                                EntityStoreSchemaFilterEffect.Exclude),
+                        },
+                    new[] { "alias1", "alias2" })
+                    .CreateWhereClause(parameters).ToString());
+
+            Assert.Equal(2, parameters.Count);
+            Assert.Equal("p0", parameters[0].ParameterName);
+            Assert.Equal("p0", parameters[0].ParameterName);
```

Comment:
@divega Was this meant to be about `p1`? And did you mean to add another `Assert.Equal()` for `p1` below?

Reply:
Yes, it looks like a copy&paste error.

Reply:
Fixed in latest commit.